source/connection-monitoring-and-pooling/tests/pool-clear-min-size.yml

```diff
@@ -0,0 +1,36 @@
+version: 1
+style: unit
+description: pool clear halts background minPoolSize establishments
+poolOptions:
+  minPoolSize: 1
+operations:
+  - name: ready
+  - name: waitForEvent
+    event: ConnectionReady
+    count: 1
+  - name: clear
+  # ensure no connections created after clear
+  - name: wait
+    ms: 200
```

Comment:
In Python our minPoolSize thread runs with a 1 second interval so this test doesn't really work. I think some drivers have a 60 second interval, should we provide some guidance in the README to reduce the frequency to ~50ms?

Reply:
Done. This is only a temporary fix for now, I think we should standardize on this across drivers, especially if the amount seems to vary by so much. Filed DRIVERS-1465 to track the follow on work for that.